### ISSUE TITLE
🐛 [BUG] - buildResolver is abstruct

### ISSUE THREAD

**Issue by kanik588** [Oksydan/falcon#152]
### Description

After installing I got these errors

Fatal Error: Class Oksydan\Module\IsThemeCore\Form\Settings\GeneralConfiguration contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (PrestaShop\PrestaShop\Core\Configuration\AbstractMultistoreConfiguration::buildResolver)

{
    "exception": {}
}

-------------------------------------------------------

Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\FatalError: "Error: Class Oksydan\Module\IsThemeCore\Form\Settings\GeneralConfiguration contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (PrestaShop\PrestaShop\Core\Configuration\AbstractMultistoreConfiguration::buildResolver)" at /home2/rumah/public_html/modules/is_themecore/src/Form/Settings/GeneralConfiguration.php line 14

{
    "exception": {}
}

### Node.js version

v16

### php version

v8

### OS and it's version

centos 7

### Required module/theme

falcon

### Reproduction steps

```bash
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
```

**Comment by Oksydan** [comment]
Hi @kanik588,

thank you for your report. Please next tile fill issue title.

What is your version of `is_themecore` module?

**Comment by kanik588** [comment]
all modules are latest version.
Prestashop 8

**Comment by Oksydan** [comment]
I didn't reproduce problem with latest version of `is_themecore` module. 
In recent version `buildResolver` method has been added to `GeneralConfiguration` - https://github.com/Oksydan/is_themecore/blob/main/src/Form/Settings/GeneralConfiguration.php#L116-L123
This bug might be valid if you install is_themecore with version below 3.0.0 for prestashop 8.0.0. 

> Classes extending AbstractMultistoreConfiguration are now forced to use Symfony’s OptionsResolver. buildResolver method is mandatory and returns an OptionsResolver.

It is one of breaking change that has been made in prestashop 8.0. You can read about it here https://devdocs.prestashop-project.org/8/modules/core-updates/8.0/#changes

**Comment by kanik588** [comment]
![photo_2022-12-08_18-50-48](https://user-images.githubusercontent.com/52570235/206484752-b5897042-7de8-4f12-a61f-0189492469a5.jpg)
for sure its version 3 of is_thecore

**Comment by Oksydan** [comment]
Can you confirm that `buildResolver` method exists in `\src\Form\Settings\GeneralConfiguration`?

**Comment by kanik588** [comment]
yes its there 
![photo_2022-12-08_19-03-14](https://user-images.githubusercontent.com/52570235/206487802-0455854b-6104-4e13-bd85-55436625aa9a.jpg)

After recopying is_themecore there isnt any fatal error but some strange thing occured.

1. after changing theme to falcon still classic theme is current theme.
2. shop theme changed but  theme not fully loaded
![Screenshot 2022-12-08 at 19-07-48 Rumah](https://user-images.githubusercontent.com/52570235/206489277-d0495972-ea77-4034-95c8-57196da55fa5.png)

**Comment by Oksydan** [comment]
Did you follow installation process and build theme assets.

**Comment by kanik588** [comment]
![photo_2022-12-08_19-18-36](https://user-images.githubusercontent.com/52570235/206493105-647591a5-bf7a-422a-9a5d-7ae3992e3e83.jpg)
 I did it again but I got  some errors.
I will try to fix font loading problem myself and let you know the problem.
thank you for your help

**Comment by Oksydan** [comment]
I isn't error but deprecation warring in bootstrap. Problem will be resolved by migrating to bootstrap 5 in version 4.0.0.

**Comment by kanik588** [comment]
all problems now solved.
in "project-init" I dident added last slash. "/"

![Screenshot 2022-12-08 at 20-10-56 GTmetrix Performance Report](https://user-images.githubusercontent.com/52570235/206511394-844fa34c-159a-4835-974a-0287e9277cf5.png)

very good result.
